languages/zCEE2.groovy

```diff
@@ -123,11 +123,14 @@ sortedList.each { buildFile, inputType ->
         StringBuffer shellError = new StringBuffer()
 
         String JAVA_HOME = props.getFileProperty('zcee2_JAVA_HOME', buildFile)
```

Comment:
@M-DLB .
Can we probably use `javaHome` as a variable name, instead of the environment variable name?